package-lock.json

```diff
@@ -11681,6 +11691,9 @@
         "glob": "^7.1.3"
       }
     },
+    "rosetta-client": {
```

Comment:
not sure this block is needed because there is already `"packages/rosetta-client"`

Reply:
It will change, yes